ui/react-components/components/ColorPicker/ColorPicker.test.jsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import { render, fireEvent } from "@testing-library/react";
+import ColorPicker from "./ColorPicker";
+import {
+  getByLabelText,
+  getByText,
+  getByTestId,
+  queryByTestId,
+  wait
+} from "@testing-library/dom";
+import "@testing-library/jest-dom/extend-expect";
+import ReactTestUtils from "react-dom/test-utils";
+
+describe("<ColorPicker />", () => {
+  it("should render ColorPicker component ", () => {
+    const { container } = render(<ColorPicker />);
+    expect(container.querySelector(".colorPickerWrapper")).not.toBeNull();
+  });
+
+  it("Should toggle the display of color picker on click the picker", () => {
```

Comment:
Not able to get the intention.
If makes sense, `Should toggle the color picker palette's visibility on click of color picker icon/dropdown/option.`

Reply:
Done